nbqa/output_parser.py

```diff
@@ -68,6 +68,15 @@ def _get_pattern(
             ),
         ]
 
+    if command.startswith("ruff"):
+        return [
+            (
+                rf"(?<=--> {re.escape(relative_path)}:)\d+"
```

Comment:
I am a bit confused what's happening here.

I tried `rf"(?<=\s+--> {re.escape(relative_path)}:)\d+"` but it fails because re.PatternError: look-behind requires fixed-width pattern